src/components/focus-manager/helpers/index.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Please refer to the terms of the license
+ * agreement.
+ *
+ * (c) 2024 Feedzai, Rights Reserved.
+ */
+import { FocusManagerProps } from "../types";
+import { Options } from "focus-trap";
+
+/**
+ * Sets the initial focus element.
+ */
+export function getInitialFocus(
+	autoFocus: FocusManagerProps["autoFocus"],
+	initialFocus: Options["initialFocus"],
+) {
+	if (autoFocus === true) {
```

Comment:
Isn't this enough?
```js
if (autoFocus) {
```

Reply:
I've changed the implementation of this to be more clear:
```ts
/**
 * Sets the initial focus element.
 */
export function getInitialFocus(
	autoFocus: FocusManagerProps["autoFocus"],
	initialFocus: Options["initialFocus"],
) {
	// If true, resumes the normal behaviour
	if (autoFocus === true) {
		return undefined;
	}

	// If false, prevents the autoFocus from happening
	else if (autoFocus === false) {
		return false;
	}

	// If is nil, uses the value defined in the `initialFocus` param,
	// which if also nil, resumes the normal behaviour
	return initialFocus;
}
```